MetaMask/metamask-mobile
# fix: Disable SES on iOS

## **Description**

This PR disables SES due to crashing issues associated with enabling SES. 

## **Related issues**

Fixes: #8054 

## **Manual testing steps**

1. Got to optimism
2. Trigger a transaction and observe crash when hitting confirm

## **Screenshots/Recordings**



### **Before**



### **After**



## **Pre-merge author checklist**


## **Pre-merge reviewer checklist**
